--- config.def.h
+++ config.h
@@ -2,13 +2,13 @@
 /* Default settings; can be overriden by command line. */
 
 static int top              = 1;                       /* -b option; if 0, appear at bottom */
-static const char *fonts[]  = { "monospace:size=10" }; /* -f option overrides fonts[0] */
+static const char *fonts[]  = { "Monaspice Kr Nerd Font:size=13" }; /* -f option overrides fonts[0] */
 static const char *prompt   = NULL;                    /* -p option; prompt to the left of input field */
 static uint32_t colors[][2] = {
 	/*               fg         bg          */
-	[SchemeNorm] = { 0xbbbbbbff, 0x222222ff },
-	[SchemeSel]  = { 0xeeeeeeff, 0x005577ff },
-	[SchemeOut]  = { 0x000000ff, 0x00ffffff },
+	[SchemeNorm] = { 0xcdd6f4ff, 0x1e1e2eff },
+	[SchemeSel]  = { 0xf9e2afff, 0x1e1e2eff },
+	[SchemeOut]  = { 0xf38ba8ff, 0x1e1e2eff },
 };
 
 /* -l option; if nonzero, use vertical list with given number of lines */
